### PR TITLE
1. Adds all splash page content

### PR DESCRIPTION
**Pull request by georgeweiler**

### CODE REVIEW

File: src/components/PageSection.tsx
Comment (r-czajkowski) [inline src/components/PageSection.tsx:15]: Can we add `as="section"`?

File: src/components/Buttons/DiscordButton.tsx
Comment (r-czajkowski) [inline src/components/Buttons/DiscordButton.tsx:8]: Looks like we can use `ExternalButtonLink` component. Also, we repeat almost the same for each social media site. Maybe we can create something like:
```
const SocialMediaButton = ( { socialMediType: DISCORD|GITHUB|TELEGRAM } ) => {
 return <render the correct button based on the `socialMediType`>
}
```

and then in other component:
```
<SocialButton socialMediType={DISCORD} />
```

Just a thought but it won't probably work well with the cms until we add support for `mdx` files.

File: src/components/Buttons/GithubButton.tsx
Comment (r-czajkowski) [inline src/components/Buttons/GithubButton.tsx:8]: Same as above.

File: src/components/Buttons/TelegramButton.tsx
Comment (r-czajkowski) [inline src/components/Buttons/TelegramButton.tsx:8]: Same as above.

File: src/pages/Splash/HarnessThePower/index.tsx
Comment (r-czajkowski) [inline src/pages/Splash/HarnessThePower/index.tsx:29]: Just flagging- we need to use `Link` component from the `gatsby` lib.

```
import { Link as GatsbyLink } from "gatsby"
```

File: src/pages/Splash/UserRoles/TakeTheQuiz.tsx
Comment (r-czajkowski) [inline src/pages/Splash/UserRoles/TakeTheQuiz.tsx:49]: Do we need this?

**Review by r-czajkowski — APPROVED** [review]
Looking good to me. Just left few minor comments that we can address in a follow-up work.

File: src/content/pages/index.md
Comment (r-czajkowski) [inline src/content/pages/index.md:77]: In this section, we don't have bg color picker. Wonder if we really need bg color picker in cms. We should ask our designers.

File: src/content/pages/index.md
Comment (r-czajkowski) [inline src/content/pages/index.md:95]: If it's a markdown we should inject the html via `dangerouslySetInnerHTML={{ __html: description }}` in component. Currently it renders as:

image

File: src/templates/home-page-old/index.tsx
Comment (r-czajkowski) [inline src/templates/home-page-old/index.tsx:14]: Let's rermove this file.

File: src/templates/home-page/index.tsx
Comment (r-czajkowski) [inline src/templates/home-page/index.tsx:38]: Based on the Figma views the `current proposals` section should be a part of the `active community` section. https://www.figma.com/file/q8Aef9yTHUipGdnqzmh6UG/Website?node-id=1111%3A16916

**Review by r-czajkowski — CHANGES_REQUESTED** [review]

**Review by r-czajkowski — COMMENTED** [review]
The images are too small in each section:
image

We probably need to create a query according to the [Gatsby Image plugin docs](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/)

**Review by r-czajkowski — COMMENTED** [review]
Images in `Harness the power...` are of low quality and they look like to big.
image

File: src/cms/gqlTypes.ts
Comment (georgeweiler) [reply src/cms/gqlTypes.ts:1]: nope, forgot to delete. Removed.

File: src/components/AppLayoutWrapper.tsx
Comment (georgeweiler) [reply src/components/AppLayoutWrapper.tsx:15]: good catch. removed.

File: src/components/AppLayoutWrapper.tsx
Comment (georgeweiler) [reply src/components/AppLayoutWrapper.tsx:19]: 👍

File: src/components/Buttons/CmsButtonLink.tsx
Comment (georgeweiler) [reply src/components/Buttons/CmsButtonLink.tsx:40]: yeah it's a bit cleaner. updated.

File: src/content/components/nav-bar.md
Comment (georgeweiler) [reply src/content/components/nav-bar.md:41]: yeah, there are a ton of links that we need to update. My preference is to list out all the links we need, double check with product & design and then use the CMS to add them all so we aren't blocked on finding the right links now.

File: src/content/pages/index.md
Comment (georgeweiler) [reply src/content/pages/index.md:14]: good call. I've added a `preTitle` field on the earn-staker branch. Could we use that? https://github.com/threshold-network/splash-page/pull/23/files#diff-cb493fc51ef33b826bceb26a12860161a9ad0d86af05e50b19b3b7b6bc5050b7R31

File: src/content/pages/index.md
Comment (georgeweiler) [reply src/content/pages/index.md:37]: I'm going to  ask the designers if we can use regular hex colors for these. I think the alpha colors are not great because they're just a opacity filter over the background. In any case, these sections have editable bg colors so I think it's okay to leave it up to the CMS users to define them at the end.

Using blackAlpha doesn't look right.

wdyt?

File: src/content/pages/index.md
Comment (r-czajkowski) [reply src/content/pages/index.md:95]: Done please see https://github.com/threshold-network/splash-page/pull/22.

File: src/content/pages/index.md
Comment (georgeweiler) [reply src/content/pages/index.md:77]: I removed it. They never asked for it, just figured it would be helpful

**Review by r-czajkowski — APPROVED** [review]
:shipit:

**Pull request merged**
